tests/integration_tests/test_mongo_task.py

```diff
@@ -502,6 +591,48 @@ def test_object_querying_mongo(
     }
 
 
+@pytest.mark.integration
+def test_get_cached_data_for_erasures(
+    integration_postgres_config, integration_mongodb_config
+) -> None:
+    privacy_request = PrivacyRequest(id=f"test_mongo_task_{random.randint(0,1000)}")
+
+    mongo_dataset, postgres_dataset = combined_mongo_postgresql_graph(
+        integration_postgres_config, integration_mongodb_config
+    )
+    graph = DatasetGraph(mongo_dataset, postgres_dataset)
+
+    access_request_results = graph_task.run_access_request(
+        privacy_request,
+        policy,
```

Comment:
Just to note that this is using the empty `policy` that's defined on line 46, rather that the `policy` or `erasure_policy` fixture. I don't think it's significant here because the erasure data will get cached regardless, so maybe that's intentional?

Reply:
Good eye - you're right that the policy isn't significant for this test, but the overlap between the empty policy and the fixture policy isn't great, I'll rename the empty policy.